Got an error when setting a default value to the Param(Boolean)

## Expected Behavior
The `Param` be settable a default value by `Default` in the Goa DSL.

## Actual Behavior
Set the `Param` to `true` by `Default`.
I got an error when `go build`, as below:
```shell
tool/cli/commands.go:239:34: cannot use true (type bool) as type string in argument to cc.Flags().StringVar
```
`tool/cli/commands.go:239` is below:
```go
	cc.Flags().StringVar(&cmd.Bool, "bool", true, ``)
```

## Steps to Reproduce the Problem (including complete and simple design if relevant)
`design.go`:
```go
package design

import (
	. "github.com/goadesign/goa/design"
	. "github.com/goadesign/goa/design/apidsl"
)

var _ = API("dummy", func() {
	Title(`Dummy API`)
})

var _ = Resource("dummy", func() {
	Action("get", func() {
		Routing(GET(""))
		Params(func() {
			Param("bool", Boolean, "", func() {
				Default(true)
			})
		})
		Response(OK)
	})
})
```

`goagen` and `go build`:
```shell
$ goagen client -d $(go list) && go build tool/dummy-cli/main.go
```

## Specifications
- goa Version: goagen v1.3.1 (3146386, 2018-04-04)
- Platform (Linux/Windows etc.): macOS 10.13.3 (17D102)
- go version go1.10.1 darwin/amd64

--- a/goagen/gen_client/cli_generator.go
+++ b/goagen/gen_client/cli_generator.go
@@ -113,6 +113,7 @@ func (g *Generator) generateCommands(commandsFile string, clientPkg string, func
 	funcs["joinRouteParams"] = joinRouteParams
 	funcs["routes"] = routes
 	funcs["flagType"] = flagType
+	funcs["defaultVal"] = defaultVal
 	funcs["cmdFieldType"] = cmdFieldTypeString
 	funcs["formatExample"] = formatExample
 	funcs["shouldAddExample"] = shouldAddExample
@@ -588,6 +589,13 @@ func flagType(att *design.AttributeDefinition) string {
 	}
 }
 
+func defaultVal(att *design.AttributeDefinition) string {
+	if att.Type.Kind() == design.IntegerKind {
+		return fmt.Sprintf("%v", att.DefaultValue)
+	}
+	return fmt.Sprintf("%q", fmt.Sprintf("%v", att.DefaultValue))
+}
+
 func shouldAddExample(ut *design.UserTypeDefinition) bool {
 	if ut == nil {
 		return false
@@ -792,14 +800,14 @@ func (cmd *{{ $cmdName }}) RegisterFlags(cc *cobra.Command, c *{{ .Package }}.Cl
 {{ end }}{{ $pparams := defaultRouteParams .Action }}{{ if $pparams }}{{ range $pname, $pparam := $pparams.Type.ToObject }}{{ $tmp := goify $pname false }}{{/*
 */}}{{ if not $pparam.DefaultValue }}	var {{ $tmp }} {{ cmdFieldType $pparam.Type false }}
 {{ end }}	cc.Flags().{{ flagType $pparam }}Var(&cmd.{{ goify $pname true }}, "{{ $pname }}", {{/*
-*/}}{{ if $pparam.DefaultValue }}{{ printf "%#v" $pparam.DefaultValue }}{{ else }}{{ $tmp }}{{ end }}, ` + "`" + `{{ escapeBackticks $pparam.Description }}` + "`" + `)
+*/}}{{ if $pparam.DefaultValue }}{{ defaultVal $pparam }}{{ else }}{{ $tmp }}{{ end }}, ` + "`" + `{{ escapeBackticks $pparam.Description }}` + "`" + `)
 {{ end }}{{ end }}{{ $params := .Action.QueryParams }}{{ if $params }}{{ range $name, $param := $params.Type.ToObject }}{{ $tmp := goify $name false }}{{/*
 */}}{{ if not $param.DefaultValue }}	var {{ $tmp }} {{ cmdFieldType $param.Type false }}
 {{ end }}	cc.Flags().{{ flagType $param }}Var(&cmd.{{ goify $name true }}, "{{ $name }}", {{/*
-*/}}{{ if $param.DefaultValue }}{{ printf "%#v" $param.DefaultValue }}{{ else }}{{ $tmp }}{{ end }}, ` + "`" + `{{ escapeBackticks $param.Description }}` + "`" + `)
+*/}}{{ if $param.DefaultValue }}{{ defaultVal $param }}{{ else }}{{ $tmp }}{{ end }}, ` + "`" + `{{ escapeBackticks $param.Description }}` + "`" + `)
 {{ end }}{{ end }}{{ $headers := .Action.Headers }}{{ if $headers }}{{ range $name, $header := $headers.Type.ToObject }}{{/*
 */}} cc.Flags().StringVar(&cmd.{{ goify $name true }}, "{{ $name }}", {{/*
-*/}}{{ if $header.DefaultValue }}{{ printf "%q" $header.DefaultValue }}{{ else }}""{{ end }}, ` + "`" + `{{ escapeBackticks $header.Description }}` + "`" + `)
+*/}}{{ if $header.DefaultValue }}{{ defaultVal $header }}{{ else }}""{{ end }}, ` + "`" + `{{ escapeBackticks $header.Description }}` + "`" + `)
 {{ end }}{{ end }}}`
 
 const commandsTmpl = `

